neqo-transport/src/shuffle.rs

```diff
@@ -48,6 +48,9 @@ pub fn find_sni(buf: &[u8]) -> Option<Range<usize>> {
         if ext_type == 0 {
             // SNI!
             let sni_len: u16 = dec.decode_uint()?;
```

Comment:
Driveby:

```suggestion
            let sni_len = dec.decode_uint::<u16>()?;
```

I find the type annotation to be action at a distance, which makes the turbofish the more explicit option.  (I'm not myself free from this particular blemish; this is more characters, so I might have exploited type inference in the past, but no time like the present to start rectifying it :)